tools/scripts/cleanCRD.sh

```diff
@@ -3,6 +3,10 @@
 # Copyright 2022 Authors of spidernet-io
 # SPDX-License-Identifier: Apache-2.0
 
```

Comment:
I suggest to add `set -x` at the very beginning , for checking log and easily debugging